### PR TITLE
Added packets

### PR DESCRIPTION
**Pull request by Q-Master**
pure Nim declarative packes system for serializing/deserializing

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -19631,5 +19631,18 @@
...
+    "url": "https://github.com/Q-Master/packets.nim.git",
+    "method": "git",
+    "tags": [
+      "serializtion",
+      "deserialization",
+      "marshal"
+    ],
+    "description": "Declarative packets system for serializing/deserializing and marshalling",
+    "license": "MIT",
+    "web": "https://github.com/Q-Master/packets.nim.git"
```
Comment (Yardanico) [inline packages.json:19646]: You should remove .git there

File: packages.json [hunk 2]
```diff
@@ -19631,5 +19631,18 @@
     "description": "A fast and simple command line argument parser inspired by Python's argparse.",
     "license": "MIT",
     "web": "https://github.com/oliversandli/clapfn"
+  },
+  {
+    "name": "packets",
+    "url": "https://github.com/Q-Master/packets.nim.git",
```
Comment (Yardanico) [inline packages.json:19637]: No need for .git here

File: packages.json (hunk 1 above)
Comment (Q-Master) [reply packages.json]: fixed

File: packages.json (hunk 2 above)
Comment (Q-Master) [reply packages.json]: fixed

**Pull request merged**